modin/experimental/core/storage_formats/pandas/parsers.py

```diff
@@ -114,6 +114,24 @@ def parse(fname, **kwargs):
         return _split_result_for_readers(1, num_splits, df) + [length, width]
 
 
+@doc(_doc_pandas_parser_class, data_type="parquet files")
+class ExperimentalPandasParquetParser(PandasParser):
+    @staticmethod
+    @doc(_doc_parse_func, parameters=_doc_parse_parameters_common)
+    def parse(fname, **kwargs):
+        warnings.filterwarnings("ignore")
+        num_splits = 1
```

Comment:
Why 1?

Reply:
Each file is equal to one partition.

Reply:
We should probably change this in a separate PR.

Reply:
This is an opportunity for further optimization, so if necessary, yes. However it's more important to add support for different formats for now.

Reply:
File an issue for further optimization please.

Reply:
https://github.com/modin-project/modin/issues/6857